src/nxdoc/web-ui/web-ui-tutorials/web-ui-configuration-properties.md

```diff
@@ -0,0 +1,183 @@
+    <property name="org.nuxeo.web.ui.enrichers.document">permissions</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">breadcrumb</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">preview</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">favorites</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">subscribedNotifications</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">thumbnail</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">renditions</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">pendingTasks</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">runnableWorkflows</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">runningWorkflows</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">collections</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">audit</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">subtypes</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">tags</property>
+    <property name="org.nuxeo.web.ui.enrichers.document">publications</property>
+    <property name="org.nuxeo.web.ui.enrichers.blob" list="true">appLinks</property>
+
+  </extension>
+</component>
+```
+
+### Special HTTP Headers
+In a similiar way, Nuxeo also makes use of special HTTP headers with the same general purpose: having more control on REST API calls.
+Please consider [Special HTTP Headers]({{page page='special-http-headers'}}) in order to learn more about this concept and also to be aware of the available options.
+
+By default, only the `fetch.document` HTTP header is defined in order to ensure that all the properties are being fetched for a given document.
+
+```xml
+<component name="org.nuxeo.web.ui.properties.contrib">
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+
+     <!-- Properties to be fetched when loading current document, default 'properties' means all -->
+    <property name="org.nuxeo.web.ui.fetch.document" list="true">properties</property>
+    
+  </extension>
+</component>
+```
+
+### Date Formats
+Since Nuxeo Web UI is an internationalized and localized application, there is a concern with language and culture. For this reason, it is possible to configure some date and time related formats.
+
+By default, the `dateFormat` and `dateTimeFormat` are defined according to Moment.js syntax (see more [here](https://momentjs.com/)), since it is the library used by Web UI to manage time.
+Also, a `firstDayOfWeek` is considered in order to display a date picker, for example, according to the cultural needs. By not having that variable defined, it will consider the most common case that is Sunday. An example of its usage can be found on [Nuxeo Date Picker](https://github.com/nuxeo/nuxeo-elements/blob/master/ui/widgets/nuxeo-date-picker.js#L86-L100).
```

Comment:
```suggestion
Also, a `firstDayOfWeek` is considered in order to display a date picker, for example, according to cultural needs. If the variable is not customized, it will defaulto to the most common case which is Sunday. An example of its usage can be found on [Nuxeo Date Picker](https://github.com/nuxeo/nuxeo-elements/blob/master/ui/widgets/nuxeo-date-picker.js#L86-L100).
```